Index: C:/Trifon-CD-0.2/COMAP/adempiere-trunk/utils/RUN_Adempiere.sh
===================================================================
--- C:/Trifon-CD-0.2/COMAP/adempiere-trunk/utils/RUN_Adempiere.sh	(revision 1227)
+++ C:/Trifon-CD-0.2/COMAP/adempiere-trunk/utils/RUN_Adempiere.sh	(working copy)
@@ -19,9 +19,9 @@
 
 ## Check Adempiere Home
 if [ $ADEMPIERE_HOME ]; then
-  CLASSPATH=$ADEMPIERE_HOME/lib/customization.jar:$ADEMPIERE_HOME/lib/patches.jar:$ADEMPIERE_HOME/lib/Adempiere.jar:$ADEMPIERE_HOME/lib/AdempiereCLib.jar:$CLASSPATH
+  CLASSPATH=$ADEMPIERE_HOME/lib/customization.jar:$ADEMPIERE_HOME/lib/patches.jar:$ADEMPIERE_HOME/lib/CompiereJasperReqs.jar:$ADEMPIERE_HOME/lib/Adempiere.jar:$ADEMPIERE_HOME/lib/AdempiereCLib.jar:$CLASSPATH
 else
-  CLASSPATH=lib/customization.jar:lib/patches.jar:lib/Adempiere.jar:lib/AdempiereCLib.jar:$CLASSPATH
+  CLASSPATH=lib/customization.jar:lib/patches.jar:lib/CompiereJasperReqs.jar:lib/Adempiere.jar:lib/AdempiereCLib.jar:$CLASSPATH
   echo ADEMPIERE_HOME is not set
   echo   You may not be able to start Adempiere
   echo   Set ADEMPIERE_HOME to the directory of Adempiere.
Index: C:/Trifon-CD-0.2/COMAP/adempiere-trunk/utils/RUN_Adempiere.bat
===================================================================
--- C:/Trifon-CD-0.2/COMAP/adempiere-trunk/utils/RUN_Adempiere.bat	(revision 1227)
+++ C:/Trifon-CD-0.2/COMAP/adempiere-trunk/utils/RUN_Adempiere.bat	(working copy)
@@ -22,7 +22,7 @@
 
 :CHECK_ADEMPIERE
 @if not "%ADEMPIERE_HOME%" == "" goto ADEMPIERE_HOME_OK
-Set CLASSPATH=lib\customization.jar;lib\patches.jar;lib\Adempiere.jar;lib\AdempiereCLib.jar;%CLASSPATH%
+Set CLASSPATH=lib\customization.jar;lib\patches.jar;lib\CompiereJasperReqs.jar;lib\Adempiere.jar;lib\AdempiereCLib.jar;%CLASSPATH%
 set ADEMPIERE_HOME=%~dp0..
 @Echo ADEMPIERE_HOME is not set.  
 @Echo   You may not be able to start Adempiere
@@ -31,7 +31,7 @@
 @Echo     cscript WinEnv.js C:\Adempiere C:\j2sdk1.4.2_08
 @goto MULTI_INSTALL
 :ADEMPIERE_HOME_OK
-@Set CLASSPATH=%ADEMPIERE_HOME%\lib\customization.jar;%ADEMPIERE_HOME%\lib\patches.jar;%ADEMPIERE_HOME%\lib\Adempiere.jar;%ADEMPIERE_HOME%\lib\AdempiereCLib.jar;%CLASSPATH%
+@Set CLASSPATH=%ADEMPIERE_HOME%\lib\customization.jar;%ADEMPIERE_HOME%\lib\patches.jar;%ADEMPIERE_HOME%\lib\CompiereJasperReqs.jar;%ADEMPIERE_HOME%\lib\Adempiere.jar;%ADEMPIERE_HOME%\lib\AdempiereCLib.jar;%CLASSPATH%
 
 :MULTI_INSTALL
 @REM  To switch between multiple installs, copy the created Adempiere.properties file
